### ISSUE TITLE
Handle npm view failures

### ISSUE THREAD

**Issue by Arcanemagus** [Arcanemagus/check-peer-deps#6]
`npm view` can fail, this needs to be handled:

```
(node:8720) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Error: ┌─────────────────────────────────────────────────────────────┐
│                   npm update check failed                   │
│             Try running with sudo or get access             │
│            to the local update config store via             │
│ sudo chown -R $USER:$(id -gn $USER) C:\Users\abneyl\.config │
└─────────────────────────────────────────────────────────────┘
(node:8720) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by Arcanemagus** [comment]
Related to #3.

**Comment by Arcanemagus** [comment]
I think the above error was due to a missing `await` fixed in #8 as I couldn't reproduce this when implementing #10.
